docs/tofu.logging.recipes.zio2.md

```diff
@@ -82,14 +82,34 @@ will produce:
 ```json
 {
   "level": "INFO",
-  "logger_name": "MyLogger",
+  "loggerName": "MyLogger",
```

Comment:
`logger_name` is default field of logback-logstash. This spelling is using in the other examples, return back please